pypredict/trades.py

```diff
@@ -7,6 +7,17 @@
 import websockets
 from pyensign.events import Event
 from pyensign.ensign import Ensign
+from river import compose
+from river import linear_model
+from river import preprocessing
+
+
+async def handle_ack(ack):
+    ts = datetime.fromtimestamp(ack.committed.seconds + ack.committed.nanos / 1e9)
+    print(f"Event committed at {ts}")
+
+async def handle_nack(nack):
+    print(f"Could not commit event {nack.id} with error {nack.code}: {nack.error}")
```

Comment:
Thanks, probably makes more sense to have these outside of the class for right now.